src/language/branches.md

```diff
@@ -129,20 +129,22 @@ check out the patterns documentation on [Switch statements and expressions][].
 
 ### Switch expressions
 
-A `switch` expression is similar to a `switch` statement, but you can use them
-anywhere you can use an expression. They produce a value based on the expression
-body of whichever case matches.
+A `switch` _expression_ produces a value based on the expression
+body of whichever case matches. 
+You can use a switch expression wherever Dart allows expressions*, for example:
```

Comment:
Is this asterisk intentional?

```suggestion
You can use a switch expression wherever Dart allows expressions, for example:
```

Reply:
It is 😅 I thought because it says "wherever Dart allows expressions" but then there's a caveat, "not in the beginning of an expression statement". Unless "wherever Dart allows expressions" already doesn't include expression statements?